src/lib/features/tag-type/tag-types.e2e.test.ts

```diff
@@ -96,6 +115,21 @@ test('Invalid tag types gets rejected', async () => {
         });
 });
 
+test('Tag type with invalid color format gets rejected', async () => {
+    await app.request
+        .post('/api/admin/tag-types')
+        .send({
+            name: 'invalid-color-tag',
+            description: 'A tag with invalid color',
+            color: 'not-a-color',
+        })
+        .set('Content-Type', 'application/json')
+        .expect(400)
+        .expect((res) => {
+            expect(res.body.details[0].message).toMatch(/color/);
```

Comment:
weird pattern. other tests I saw were returning body from the await call and asserting on the next line instead of going down the callback rabbit hole